modern/src/map/core/MapView.js

```diff
@@ -78,6 +78,11 @@ const MapView = ({ children }) => {
   const [activeMapStyles] = usePersistedState('activeMapStyles', ['locationIqStreets', 'osm', 'carto']);
   const [defaultMapStyle] = usePersistedState('selectedMapStyle', 'locationIqStreets');
   const mapboxAccessToken = useAttributePreference('mapboxAccessToken');
+  const maxZoom = useAttributePreference('web.maxZoom');
+
+  useEffect(() => {
+    map.setMaxZoom(maxZoom);
```

Comment:
Should we do a check here to make sure there's a value? Or it doesn't matter?

Reply:
Normaly we should, to avoid any bug in the future, another remark, when you set an attribute he's not persisting, as long as you're log in, the value stay in session but it's not stored on database, and as soon as you logout, the value disappear, can you try on your side just to know if I'm the only one facing that issue, if I'm not I'll open a ticket.

Reply:
What do you mean it's not persisted? Is it not saved in the user attributes?